o-fish-ios/Views/Components/AddAttachmentsModalView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AddAttachmentsModalView.swift
+//  
+//  Created on 6/26/20.
+//  Copyright © 2020 WildAid. All rights reserved.
+//
+
+import SwiftUI
+
+struct AddAttachmentsModalView: View {
+
+    var photo: () -> Void
+    var note: () -> Void
+    var cancel: () -> Void
+
+    private enum Dimensions {
+        static let padding: CGFloat = 8.0
+        static let buttonPadding: CGFloat = 21.0
+        static let radius: CGFloat = 15.0
+        static let bottomPadding: CGFloat = 60.0
+        static let noSpacing: CGFloat = 0.0
+    }
+
+    var body: some View {
+        VStack(spacing: Dimensions.padding) {
+            Spacer()
+            VStack(spacing: Dimensions.noSpacing) {
+
+                Button(action: photo) {
+                    Spacer()
+                    Text("Photo")
+                        .foregroundColor(.blue)
+                        .padding(Dimensions.buttonPadding)
+                    Spacer()
+                }
+                    .padding(.horizontal, Dimensions.padding)
+
+                Divider()
+                Button(action: note) {
```

Comment:
make a reusable private component from 

                Button(action: photo) {
                    Spacer()
                    Text("Photo")
                        .foregroundColor(.blue)
                        .padding(Dimensions.buttonPadding)
                    Spacer()
                }

and remove copy-paste code